reedsolomon_test.go

```diff
@@ -24,9 +26,8 @@ func TestEncoding(t *testing.T) {
 		shards[s] = make([]byte, perShard)
 	}
 
-	rand.Seed(0)
```

Comment:
Please make tests deterministic again, so the same values are always used. If I get a report of a test failure, I need to be sure that I can get the same random values. 

It will of course not be random from run to run, but that is not the intention, it is to check for regressions.